shorthand/FhirDataEvaluatorIG/input/fsh/basic-measure.fsh

```diff
@@ -0,0 +1,22 @@
+Profile: FhirDataEvaluatorBasicMeasure
+Parent: http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/measure-cqfm
+Description: "The Profile for the Measures used in the Fhir Data Evaluator"
+
+* group 1..
+* group.stratifier.criteria.language = #text/fhirpath
+* group.stratifier.component.criteria.language = #text/fhirpath
+
+* group.stratifier obeys criteriaRequiresCode
+* group.stratifier obeys codeRequiresCriteria
+* group.stratifier.component obeys criteriaRequiresCode
+
+* group.population ^slicing.discriminator.type = #value
+* group.population ^slicing.discriminator.path = "code.coding.code"
+* group.population ^slicing.rules = #open
+* group.population ^slicing.ordered = false
+
+* group.population 1..1
+* group.population contains initialPopulation 1..1
+* group.population[initialPopulation].code.coding.system = "http://terminology.hl7.org/CodeSystem/measure-population"
```

Comment:
I would use `group.population[initialPopulation].code = $Alias#initial-population` and define an alias for `http://terminology.hl7.org/CodeSystem/measure-population`. No need to set the system and code separately.